screens/Meal.tsx

```diff
@@ -1,210 +1,303 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
 import {BottomTabScreenProps} from '@react-navigation/bottom-tabs';
-import React, {useState} from 'react';
+import React, {useState, useEffect} from 'react';
 import {StyleSheet} from 'react-native';
 import {RootTabList} from '../App';
-import {Box, Center, HStack, ScrollView, Text, VStack} from 'native-base';
-import _ from 'lodash';
+import {
+  Box,
+  Center,
+  HStack,
+  ScrollView,
+  Text,
+  VStack,
+  Modal,
+  Button,
+} from 'native-base';
+import _, {size} from 'lodash';
```

Comment:
```suggestion
import {chain, size, split, partition, chunk} from 'lodash';
```
[Optional] _ 라는 것이 아무 의미도 담고있지 않아서 어떤 역할을 하는 것인지 사용하는 측에서 모르기 때문에 이렇게 쓰는것이 좋을듯!

Reply:
ㅇㅋ 난 그냥 빌트인 메소드 아니고 로대쉬꺼라고 두려고 + 걍 독스 usage랑 똑같이 쓰려고 그런거였는데 암튼 반영했음

Reply:
독스는 아마 콘솔에서 똑같이 하면 된다는 것을 표현하기 위해 그렇게 한듯!
![image](https://user-images.githubusercontent.com/43872798/133934247-51be0dca-1d48-4341-82e4-8323875a7836.png)